### PR TITLE
Improve French translations

### PR DESCRIPTION
**Pull request by damsfx**

### CODE REVIEW

File: modules/backend/lang/fr/lang.php [hunk 1]
```diff
@@ -29,9 +35,24 @@
         ],
     ],
     'partial' => [
-        'not_found_name' => 'Le modèle partiel ":name" est introuvable.'
+        'not_found_name' => 'Le modèle partiel ":name" est introuvable.',
+        'invalid_name' => 'Le nom du modèle partiel est invalide: :name.',
+    ],
+    'ajax_handler' => [
+        'invalid_name' => 'Nom du gestionnaire AJAX non valide: :name.',
+        'not_found' => "Le gestionnaire AJAX ':name' n'a pas été trouvé..",
```
Comment (mjauvin) [inline modules/backend/lang/fr/lang.php]:
```suggestion
        'not_found' => "Le gestionnaire AJAX ':name' n'a pas été trouvé.",
```

File: modules/system/lang/fr/lang.php [hunk 2]
```diff
@@ -12,11 +12,6 @@
     'file' => [
         'create_fail' => "Impossible de créer le fichier : :name",
     ],
-    'page' => [
-        'invalid_token' => [
-            'label' => 'La clé de sécurité est invalide',
-        ],
-    ],
```
Comment (mjauvin) [inline modules/system/lang/fr/lang.php:19]: Why was this removed?

**Review by mjauvin — CHANGES_REQUESTED** [review]
Please address these.

File: modules/system/lang/fr/lang.php (hunk 2 above)
Comment (damsfx) [reply modules/system/lang/fr/lang.php:19]: The `page` exist twice in the file.
https://github.com/wintercms/winter/blob/b1d28d9d3ef3e5889bf93a38dfbcfbabb723f256/modules/system/lang/fr/lang.php#L15-L19
And 
https://github.com/wintercms/winter/blob/b1d28d9d3ef3e5889bf93a38dfbcfbabb723f256/modules/system/lang/fr/lang.php#L410-L423

Comment (mjauvin) [reply modules/system/lang/fr/lang.php:19]: Yes, but we're now missing the invalid_token key under page...

File: modules/backend/lang/fr/lang.php [hunk 3]
```diff
@@ -29,9 +35,24 @@
         ],
```
Comment (mjauvin) [inline modules/backend/lang/fr/lang.php:35]:
```suggestion
        ],
        'invalid_token' => [
            'label' => 'La clé de sécurité est invalide',
        ],
```

Comment (damsfx) [inline modules/backend/lang/fr/lang.php:35]: @mjauvin Why do you want to add this entry?
It doesn't exist in the EN file.

**Comment by mjauvin** [comment]
Sorry @damsfx, I was looking in the wrong file...

**Comment by mjauvin** [comment]
Thanks @damsfx !
